samples/unit-tests/bubble-legend/styles/demo.js

```diff
@@ -65,3 +65,38 @@ QUnit.test('Bubble legend ranges', function (assert) {
         'Bubble legend was properly disabled with the legend'
     );
 });
+
+QUnit.test('Bubble legend styled mode', function (assert) {
+    var chart = Highcharts.chart('container', {
```

Comment:
Long-running pre-commit tests are becoming a problem for us. Instead of creating a new chart for this test, see if you can modify one of the existing bubble-legend tests and add the assertion there.